<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalCustomer</class>
 <widget class="QWidget" name="SignalCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>散客开单  订单号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_payid">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_idcard">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份证号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>顾客姓名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idcard">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sex">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_roomchoice">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间选择:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_roomchoice">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_roomlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>开单人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_userid">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_days">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预住天数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_countname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间折扣计算:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_roomprice">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>580</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_roomdiscount">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_roomresult">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cash">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>须收现金:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineEdit_cash">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pledge">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>押金:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pledge">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalmoney">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总需收款</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_totalmoney">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_days">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_deal">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancle">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roomname">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_days_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>顾客电话:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_call">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
